src/org/labkey/test/tests/component/GridPanelViewTest.java

```diff
@@ -1260,6 +1263,48 @@ public void testRemoveAllFields()
 
     }
 
+    @Test
```

Comment:
This coverage is on QueryGrid views, if you think it would be helpful to do the same for DataRegion views I'm happy to add